tests/test_unittest/test_gtc/gtir_utils.py

```diff
@@ -0,0 +1,20 @@
+from gt4py.gtc.gtir import CartesianOffset, FieldAccess, TernaryOp, Expr
+from gt4py.gtc.common import DataType
+
+
+class DummyExpr(Expr):
+    """Fake expression for cases where a concrete expression is not needed."""
+
+    dtype: DataType = DataType.FLOAT32
+
+
+class FieldAccessBuilder:
+    def __init__(self, name) -> None:
+        self.node = FieldAccess(name=name, offset=CartesianOffset.zero())
+
+    def offset(self, offset: CartesianOffset) -> "FieldAccessBuilder":
+        self.node = FieldAccess(name=self.node.name, offset=offset)
+        return self
+
+    def __call__(self) -> FieldAccess:
```

Comment:
Good to use `__call__()` instead of `build()` or uncommon?

Reply:
It is fine, but consider the following pattern:

```python
foo = (
    XYBuilder()
    .this_attr(value)
    .append_that(item)
    .append_that(other_item)
    ...
).build()
```

May be clearer than

```python
foo = (
    XYBuilder()
    .this_attr(value)
    .append_that(item)
    .append_that(other_item)
    ...
)()
```